src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write code here
+const employees = [...document.querySelectorAll('li')];
+
+const dataAtributToNumber = (salary) => {
+  return salary.replace(/\D/g, '');
+};
+
+const sortList = (list) => {
+  list
+    .sort(
+      (value1, value2) =>
+        dataAtributToNumber(value2.dataset.salary) -
+        dataAtributToNumber(value1.dataset.salary),
+    )
+    .forEach((employee, i) => list[i].parentElement.appendChild(employee));
+};
+
+const getEmployees = (list) => {
+  list.forEach((employee) => {
+    return {
+      name: employee.innerText,
+      position: employee.dataset.position,
+      salary: dataAtributToNumber(employee.dataset.salary),
+      age: +employee.dataset.age,
+    };
+  });
+};
```

Comment:
The `getEmployees` function does not return any value. It seems you intended to return an array of employee objects. However, using `forEach` does not return a new array. Use `map` instead, which transforms each item in the array and returns a new array.